lib/definitions/errors.js

```diff
@@ -8,9 +8,17 @@ module.exports = {
   EINVALIDCHANGELOGFILE: ({changelogFile}) => ({
     message: 'Invalid `changelogFile` option.',
     details: `The [changelogFile option](${linkify(
-      'README.md#changelogfile'
+      'README.md#options'
```

Comment:
This fix and the following one (`assets` => `changelogFile `) are unrelated to this feature. Would you mind moving them in a different commit (that can be on the same PR). Something like ``fix: correct link and description in `EINVALIDCHANGELOGFILE` error``